app.js

```diff
@@ -30,12 +31,47 @@
       this.$('.query.date').datepicker({ dateFormat: "yy-mm-dd" });
     }),
 
+    // Toggles acending/decending order of the column header clicked
+    sortTable: function(event) {
+      var $tr = this.$(event.target);
+      var position = $tr.index();
+      var $tableBody = $tr.closest('table').find('tbody');
+      var greaterThan,
+          lessThan;
+
+      $tr.addClass('sorted');
+      $tr.siblings().removeClass('sorted ascending');
+      $tr.toggleClass('ascending');
+
+      if ( $tr.hasClass('ascending') ) {
+        greaterThan = 1;
+        lessThan = -1;
+      } else {
+        greaterThan = -1;
+        lessThan = 1;
+      }
+
+      var newList = $tableBody.find('tr').sort( function(a,b) {
+        if ( this.$(a).find('td:eq(' + position + ')').text() > this.$(b).find('td:eq(' + position + ')').text() ){
```

Comment:
It looks like you may not need greaterThan and lessThan, you can just subtract the values from the two positions.
For the descending order, you could inverse the order of the elements in the subtraction.